src/sentry/grouping/enhancer/__init__.py

```diff
@@ -201,9 +202,7 @@ def apply_rust_enhancements(
         return None
 
     try:
-        use_rust_enhancements = random.random() < options.get(
-            "grouping.rust_enhancers.modify_frames_rate"
-        )
+        use_rust_enhancements = in_random_rollout("grouping.rust_enhancers.modify_frames_rate")
```

Comment:
NOTE to self: this shipped and I should just make this the default, same down below.